pkg/sfu/buffer/buffer.go

```diff
@@ -109,6 +112,7 @@ func NewBuffer(ssrc uint32, vp, ap *sync.Pool) *Buffer {
 		mediaSSRC:   ssrc,
 		videoPool:   vp,
 		audioPool:   ap,
+		pliThrottle: 500e6,
```

Comment:
nit: 500 * time.Millisecond for readability

Reply:
It is a type issue. `500 * time.Millisecond` will be `time.Duration` where as this is type `int64`. I know that `time.Duration` is just an `int64`, but these casts make me nervous :-). Will do the casting for now.